dash/dash.py

```diff
@@ -832,7 +825,8 @@ def duplicate_check():
                 ).replace('    ', '')
             raise exceptions.DuplicateCallbackOutput(msg)
 
-    def _validate_callback_output(self, output_value, output):
+    @staticmethod
+    def _validate_callback_output(output_value, output):
```

Comment:
I guess `#pylint: disable=no-self-use` was masking this for the whole rest of the class...